src/PrestaShopBundle/Security/Admin/EmployeeProvider.php

```diff
@@ -27,27 +27,38 @@
 namespace PrestaShopBundle\Security\Admin;
 
 use Access;
+use Context;
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
 use Psr\Cache\CacheItemPoolInterface;
+use Psr\Cache\InvalidArgumentException;
 use Symfony\Component\Security\Core\Exception\UnsupportedUserException;
 use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\User\UserProviderInterface;
 
 /**
  * Class EmployeeProvider To retrieve Employee entities for the Symfony security components.
+ *
+ * @deprecated since 1.7.7 - use \PrestaShopBundle\Security\Admin\UserProvider instead.
  */
 class EmployeeProvider implements UserProviderInterface
 {
     const ROLE_EMPLOYEE = 'ROLE_EMPLOYEE';
```

Comment:
Isn't this constant duplicated?

Reply:
it is, but this service is deprecated so I thought using the constant from this service wouldn't be a good idea